openwisp_notifications/templates/admin/openwisp_users/user/change_form_object_tools.html

```diff
@@ -0,0 +1,8 @@
+{% extends "admin/change_form_object_tools.html" %}
+
+{% load i18n admin_urls %}
+
+{% block object-tools-items %}
+  <li><a class="button" href="{% url 'notifications:user_notification_preference' object_id %}">Notification Preferences</a></li>
```

Comment:
Let's only show this button on staff users. Notification preferences are not created for non-staff users.

Reply:
Updated.